extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -97,15 +97,16 @@ export class SchemaCompareOptionsDialog {
 	private async reset(): Promise<void> {
 		let service = (vscode.extensions.getExtension(mssql.extension.name).exports as mssql.IExtension).schemaCompare;
 		let result = await service.schemaCompareGetDefaultOptions();
+		result.defaultDeploymentOptions = this.schemaComparison.ConvertObjectToMapTable(result.defaultDeploymentOptions);
 		this.optionsModel.deploymentOptions = result.defaultDeploymentOptions;
 		this.optionsChanged = true;
 
 		// This will update the Map table with default values
-		this.optionsModel.InitializeUpdateOptionsMapTable();
+		this.optionsModel.UpdateOptionsMapTable();
 
 		await this.updateOptionsTable();
 		this.optionsFlexBuilder.removeItem(this.optionsTable);
-		this.optionsFlexBuilder.insertItem(this.optionsTable, 0, { CSSStyles: { 'overflow': 'scroll', 'height': '65vh' } });
+		this.optionsFlexBuilder.insertItem(this.optionsTable, 0, { CSSStyles: { 'overflow': 'scroll', 'height': '65vh', 'padding-top': '2px' } });
```

Comment:
please add screenshot of what these UI changes looks like before and after

Reply:
in the future, it would be better for UI changes like this to be in a separate PR.

Reply:
I'm going to remove this change for now as we have other CSS changes: but this change is for adding little gap between the table header and the first option.
![image](https://user-images.githubusercontent.com/74571829/176269394-609a96e4-4277-475e-8cde-a2c3e63ffb19.png)